pkg/server/server_manifest_test.go

```diff
@@ -1105,8 +1103,8 @@ func getValidIssuanceTemplateRequest(m manifest.CredentialManifest, issuerDID *d
 					Schema:                    createdSchema.ID,
 					CredentialInputDescriptor: "test-id",
 					Data: issuing.ClaimTemplates{
-						"firstName": "$.credentialSubject.firstName",
-						"lastName":  "$.credentialSubject.lastName",
+						"firstName": "$.vc.credentialSubject.firstName",
+						"lastName":  "$.vc.credentialSubject.lastName",
```

Comment:
we were generating JWT-VCs but treating them as LD VCs. the path was incorrect since it pointed to a non-existant path within the cred.

Reply:
`issuing.ClaimTemplates` are associated to a `CredentialInputDescriptor`. This implies that you're already within a credential. As such, you shouldn't need to go into the `.vc` field within the JSON path. What am I missing?

Since this is a refactor, I wouldn't expect it to change any behaviors.

Reply:
updated and fixed